contrib/loaddata.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ -z ${HURUMAP_APP+x} ]; then HURUMAP_APP='hurumap_land'; fi
+if [ -z ${HURUMAP_APP+x} ]; then HURUMAP_APP='hurumap_ng'; fi
```

Comment:
Don't change this. I think `hurumap_land` is just a fallback in case `HURUMAP_APP` is not defined.